### ISSUE TITLE
chore(rln-relay): Allow running chat2/wakunode2 with a set of credentials

### ISSUE THREAD

**Issue by rymnc** [waku-org/nwaku#1449]
### Background
Right now, the only way to run chat2/wakunode2 with pre-defined credentials is to use a keyfile, that was introduced in https://github.com/waku-org/nwaku/pull/1285.

However, for quick prototyping/for those who do not want to use a keyfile, but would like to re-use credentials, we should allow setting the idCommitment, idKey and index as command line args when running the above binaries. 
 

### Details
Use command line credentials as the least prioritised credential input method. Some work has been done in the `rln-interep-poc` branch, but this feature would most likely come after the refactor planned.

### Acceptance criteria
- [ ] Allow setting idCommitment, idKey and index in command line args while running `wakunode2`/`chat2`

**Comment by vpavlin** [comment]
@rymnc Is this issue still valid?

**Comment by rymnc** [comment]
Yes, we do not support configuring rln credentials via cli/config file yet, we can prioritize this since it is required for the simulations we plan to run. 
thanks for the ping!
